server/events.go

```diff
@@ -1064,6 +1075,38 @@ func (s *Server) initEventTracking() {
 	}
 }
 
+// UserInfo returns basic information to a user about bound account and user permissions.
+// For account information they will need to ping that separately, and this allows security
+// controls on each subsystem if desired, e.g. account info, jetstream account info, etc.
+type UserInfo struct {
+	UserID      string       `json:"user"`
+	Account     string       `json:"account"`
+	Permissions *Permissions `json:"permissions,omitempty"`
```

Comment:
Also include Time of Day restrictions and CIDR block restrictions?  Expiration might be useful here too. wdyt?

Reply:
Let me add expiration, I think we hold on the others since that could open up a bunch of options on what is reported that blurs between user and accounts.

Will add expiration, that is a good add for sure.